### PR TITLE
Evo 1158 farm detail widget

### LINKED ISSUE #1158: Display all the farm data in farm details widget
Display all the extra farm related fields on the farm group about page

- [ ] Open to the public?
- [ ] Hours
- [ ] Products
- [ ] Practices
...

### PR DESCRIPTION
**Pull request by thomasgwatson**
closes #1158

### CODE REVIEW

**Comment by thomasgwatson** [comment]
Mission + Details

image

**Comment by thomasgwatson** [comment]
Details expanded

image

**Comment by thomasgwatson** [comment]
Address and Hours

image

**Comment by thomasgwatson** [comment]
Once the at-a-glance widget and collaborate widget are sorted out, this will need a styling pass from Aaron

File: src/store/selectors/farmExtensionSelectors.js [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+/*
+  Hylo sub-profile specific selectors
+*/
+
+export function getHyloSpecificFarmSchemaData (farmData) {
+  if (!farmData || !farmData.flexible || !farmData.flexible.hylo) return {}
+  return farmData.flexible.hylo
+}
+
+export function getOpenToPublic (group) {
```
Comment (tibetsprague) [inline src/store/selectors/farmExtensionSelectors.js:290]: note: might move some or all of these to the regular group.settings so we dont have to load the extension data to look this up

**Review by tibetsprague — APPROVED** [review]

**Pull request merged**